SecretSanta/test/SecretSanta.Api.Tests/Controllers/BaseApiControllerTests.cs

```diff
@@ -48,7 +48,7 @@ public async Task Get_WhenEntityDoesNotExist_ReturnsNotFound()
             TService service = new TService();
             BaseApiController<TDto, TInputDto> controller = CreateController(service);
 
-            IActionResult result = await controller.Get(1);
+            var result = await controller.Get(1);
 
             Assert.IsTrue(result is NotFoundResult);
```

Comment:
```suggestion
            Assert.IsTrue(result.Result is NotFoundResult);
```
This should fix the failing tests.